tests/testthat/test-chunked_media.R

```diff
@@ -0,0 +1,77 @@
+# Tests from https://gist.github.com/dieghernan/824425b814bbb9527db5085d6dacb157
+
+test_that("mp4 10MG works", {
```

Comment:
What's the intent of these tests? They don't appear to contain any expectations.

Reply:
Store them and check they didn't produce any error. I'll add the expectation `expect_error(..., NA)` to convey that